docs/guide/siteaccess.md

```diff
@@ -37,6 +37,15 @@ you need to create your own admin SiteAccesses and add them to this group. In ca
 ## Configuring SiteAccesses
 
 You configure SiteAccess in your config files (e.g. `ezplatform.yml`) under the `ezpublish.siteacess` keys.
+
+!!! tip
+    
+    If you encounter issues when configuring SiteAccess or want to check that system picks up the right value, use the following command:
+    
+    `bin/console [—-siteaccess=<SA>] ezplatform:debug:config-resolver <param.name>`
+    
+    The command also enables you to test how different parameters work when configuring [scopes](#scope). 
```

Comment:
This is like a tip within a tip, do we have a way to signify that?

Or should we maybe just prefix with something like `Advance Tip:` or `Advance Usage:` ?

Reply:
Will see what can we do about it. ;)